log2mission.py

```diff
@@ -31,6 +31,14 @@ def write_mission(lat, lon, alt):
 
 if __name__ == "__main__":
     filename = sys.argv[1]
+    
+    eps = 1.3 #default epsilon value
+    
+    if len(sys.argv) >= 3:
+        assert isinstance(float(sys.argv[2]), float), "Epsilon argument should be a integer"
```

Comment:
The error message is not exactly correct. The number can have a decimal place, meaning it must not be an integer. Maybe "Epsilon most be a positive number"?